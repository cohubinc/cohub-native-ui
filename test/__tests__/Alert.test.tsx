import React from "react";
import { mount, ReactWrapper } from "enzyme";

import { Alert } from "dist";

let wrapper: ReactWrapper;
beforeEach(() => {
  wrapper = mount(<Alert> Some Alert </Alert>);
});
it("matches snapshot", async () => {
  expect(wrapper).toMatchSnapshot();
});

it("renders its children", async () => {
  expect(wrapper.text().trim()).toBe("Some Alert");
});
