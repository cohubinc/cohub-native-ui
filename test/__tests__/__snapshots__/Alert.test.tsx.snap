// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Alert>
  <ScrollIntoView
    traceProp=" Some Alert "
  >
    <div
      className=" w-100"
    >
      <Fade
        duration={150}
      >
        <Transition
          duration={150}
          entered={
            Object {
              "opacity": 1,
            }
          }
          entering={
            Object {
              "opacity": 0,
            }
          }
          show={true}
          start={
            Object {
              "opacity": 0,
            }
          }
          transitionProperty="opacity"
        >
          <Transition
            appear={true}
            enter={true}
            exit={true}
            in={true}
            mountOnEnter={true}
            onEnter={[Function]}
            onEntered={[Function]}
            onEntering={[Function]}
            onExit={[Function]}
            onExited={[Function]}
            onExiting={[Function]}
            timeout={150}
            unmountOnExit={true}
          >
            <div
              key="0/.0"
              style={
                Object {
                  "backgroundColor": "#f8f8f9",
                  "border": "none",
                  "borderRadius": 4,
                  "boxShadow": "inherit",
                  "fontSize": "12px",
                  "opacity": 0,
                  "padding": "16px 20px",
                  "transition": "all 150ms ease-in-out",
                  "transitionProperty": "opacity",
                }
              }
            >
              <div
                className=""
              >
                <Component
                  style={
                    Object {
                      "color": "rgba(0,0,0,.87)",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "fontSize": 12,
                        },
                        Object {
                          "color": "rgba(0,0,0,.87)",
                        },
                        Object {
                          "fontWeight": "400",
                        },
                        Object {
                          "color": "hsl(0, 2%, 22%)",
                        },
                        Object {
                          "fontFamily": "Helvetica",
                        },
                      ]
                    }
                  >
                    <div
                      className="css-text-901oao"
                      dir="auto"
                      style={
                        Object {
                          "color": "rgba(57,55,55,1.00)",
                          "fontFamily": "Helvetica",
                          "fontSize": "12px",
                          "fontWeight": "400",
                        }
                      }
                    >
                       Some Alert 
                    </div>
                  </Text>
                </Component>
              </div>
            </div>
          </Transition>
        </Transition>
      </Fade>
    </div>
  </ScrollIntoView>
</Alert>
`;
